radius/mods-enabled/rest

```diff
@@ -10,6 +10,8 @@ rest {
     logging_api_base_url = "$ENV{LOGGING_API_BASE_URL}"
 
     authorize {
+        rewrite_called_station_id
+        rewrite_calling_station_id
```

Comment:
Should we be calling these functions before post-auth logs the request?

Reply:
FreeRADIUS recommends it in the authorize section.
```
#  Add "rewrite_called_station_id" in the "authorize" and "preacct" sections.
# Add "rewrite_calling_station_id" in the "authorize" and  "preacct" sections.
```

I also followed the convention of the other filter we have here: https://github.com/alphagov/govwifi-frontend/blob/master/radius/sites-enabled/inner-tunnel#L3

Which makes me think this could live in any of the `authorize` blocks:
- `radius/mods-enabled/rest`
- `radius/sites-enabled/default`, or
- `radius/sites-enabled/inner-tunnel`